docs/reference-guides/data/data-core-editor.md

```diff
@@ -435,6 +435,10 @@ _Related_
 
 -   getInserterItems in core/block-editor store.
 
+### getInserterSearchInputRef
+
+Undocumented declaration.
+
```

Comment:
Let's make this a private selector

Reply:
I believe we can move this all into the block editor store. We don't need this in editor at all.